### DISCUSSION TITLE
第6节课作业（20班）

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#474] [作业]
作业这里哦

**Comment by liu673** [comment]
https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_6/lesson_6_homework.md

**Comment by deyouzuishuai** [comment]
https://zhuanlan.zhihu.com/p/679628950

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/679824362

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135878525
